### DISCUSSION TITLE
Alterando .htacess para ocultar diretórios e arquivos

### DISCUSSION THREAD

**Discussion by RFS-Solutions** [RamonSilva20/mapos#2596] [General]
Boa tarde,

Para podermos alterar o .htaccess e ocultar os diretórios e arquivos, como podemos proceder?

Fiz uns testes com o arquivo .htaccess do diretório raiz, mas não funcionou. Ai pesquisando, vi que tem outros .htaccess espalhados no MAPs, mas mesmo alterando eles, continua sem reescrever/ocultar.

Seria para deixar apenas www.dominio.com/

Alguém poderia me ajudar?

**Comment by willph** [comment]
Usa apache, nginx ou traefik na sua hospedagem?

**Reply by RFS-Solutions** [reply]
Apache

**Comment by RFS-Solutions** [comment]
Alguém saberia informar como/qual .htaccess tem de ser configurado na pasta do sistema?

Agradeço

**Reply by GhosT-Exe1** [reply]
tenta o que fica na pagina inicial e coloca 

   #Remova espaços em branco URL
    RewriteCond %{REQUEST_URI} ^(.*)\s(.*)$ 
    RewriteRule ^(.*)\s(.*)$ /%1/%2 [R=301,L]

    #Remova o index.php da URL
    RewriteCond %{REQUEST_FILENAME} !-f
    RewriteCond %{REQUEST_FILENAME} !-d
    RewriteRule ^(.*)$ index.php/$1 [L]
